packages/lesswrong/server/scripts/testPostDescription.ts

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-console */
+import { getPostDescription } from "../../components/posts/PostsPage/PostsPage";
+import { Posts } from "../../lib/collections/posts";
+import Revisions from "../../lib/collections/revisions/collection";
+import { Vulcan } from "../vulcan-lib";
+
+
+/** For visually inspecting that our descriptions match the post content well */
+const run = async () => {
```

Comment:
You're welcome to have a go if you want:

```
❯ ./scripts/serverShellCommand.sh 'Globals.testPostDescription()'
```

Reply:
Nice!